### PR TITLE
Sync static template

### PR DESCRIPTION
**Pull request by 3noch**
And don't render CSRF/hidden fields in static renderer.

### CODE REVIEW

File: keg_elements/templates/keg_elements/forms/horizontal_static.html [hunk 1]
```diff
@@ -9,28 +9,30 @@
...
-        {% endif %}
-        <div class="labeled-group">
-            <p id="{{ field.id }}" class="form-control-static">
-                {%- if field.type == 'SelectField' -%}
-                    {{ field.selected_choice_label }}
-                {%- else -%}
-                    {{ field.data }}
-                {%- endif -%}
-            </p>
+    {% if field.type != 'HiddenField' and field.type !='CSRFTokenField' %}
```
Comment (3noch) [inline keg_elements/templates/keg_elements/forms/horizontal_static.html]: Moved this `if` up a line, which indented this entire block.

File: keg_elements/templates/keg_elements/forms/horizontal.html [hunk 2]
```diff
@@ -176,7 +176,6 @@
     </form>
 {%- endmacro %}
 
-
```
Comment (nZac) [inline keg_elements/templates/keg_elements/forms/horizontal.html]: Random -?

Comment (nZac) [inline keg_elements/templates/keg_elements/forms/horizontal.html]: ohhh, I see, this is the diff marker... I am going crazy

Comment (3noch) [inline keg_elements/templates/keg_elements/forms/horizontal.html]: Wow, that **is** pretty confusing.

**Pull request merged**